loops.js

```diff
@@ -47,3 +64,18 @@
 // >> 7
 // >> 9
 // ==========================================
+var total = 9;
```

Comment:
In de opdracht stond niet dat er een totaal berekend moest worden, denk dat dit een test voor jezelf was?
Dan had je dus ook geen var hoeven gebruiken en gewoon gelijk met de for loop kunnen beginnen. 
Let er ook op dat je var eigenlijk niet gebruikt en beter kunt kiezen voor const of let.